front/components/tables/TablePicker.tsx

```diff
@@ -220,15 +220,15 @@ export default function TablePicker({
                     handleLoadNext(nextPageCursor);
                   }}
                   hasMore={!!nextPageCursor}
-                  isValidating={isTablesLoading || isDebouncing}
-                  isLoading={isTablesLoading || isDebouncing}
-                >
-                  {(isTablesLoading || isDebouncing) && !allTablesMap.size && (
+                  showLoader={
+                    !allTablesMap.size && (isTablesLoading || isDebouncing)
```

Comment:
@JulesBelveze do we want to show this loader only when there is no result yet? 🤔 (I wonder if we also want to show when we are fetching next pages too)